soc/arm/nxp_imx/rt/Kconfig.soc

```diff
@@ -111,6 +140,9 @@ config INIT_SYS_PLL
 config INIT_USB1_PLL
 	bool "Initialize USB1 PLL"
 
+config ETH_MCUX_0
```

Comment:
This should be `INIT_ENET_PLL` instead. `ETH_MCUX_0` is already defined in `drivers/ethernet/Kconfig.mcux`

Reply:
updated, as well as in soc.c. There was a patch to add ENET PLL initialization using this macro in soc.c

Reply:
Thanks for the change. I can confirm that it now builds fine for both `mimxrt1020_evk`  and `mimxrt1050_evk`. That said with that change, the ENET PLL will not be enabled anymore for the `mimxrt1050_evk` when the Ethernet driver is enabled. Is it needed to get the Ethernet driver working? I personally can't test.